tfhe/examples/utilities/generates_test_keys.rs

```diff
@@ -19,9 +20,10 @@ use tfhe::shortint::parameters::parameters_wopbs_message_carry::{
 };
 use tfhe::shortint::parameters::{
     ClassicPBSParameters, WopbsParameters, ALL_MULTI_BIT_PARAMETER_VEC, ALL_PARAMETER_VEC,
-    PARAM_MESSAGE_1_CARRY_1_KS_PBS, PARAM_MESSAGE_2_CARRY_2_KS_PBS, PARAM_MESSAGE_3_CARRY_3_KS_PBS,
-    PARAM_MESSAGE_4_CARRY_4_KS_PBS,
+    PARAM_MESSAGE_1_CARRY_1_KS_PBS, PARAM_MESSAGE_2_CARRY_2_KS_PBS, PARAM_MESSAGE_2_CARRY_3_KS_PBS,
```

Comment:
we add the 2_3 for which reason ? It seems to me we have coverage params for that ?